src/GraphQL/Types/Scalars/IntGraphType.cs

```diff
@@ -1,13 +1,16 @@
 using GraphQL.Language.AST;
+using GraphQL.Utilities;
 
 namespace GraphQL.Types
 {
     /// <summary>
-    /// The Int scalar type represents a signed 32‐bit numeric non‐fractional value.
+    /// The Int scalar type represents a signed 32‐bit numeric non‐fractional value. It is one of the five built-in scalars.
+    /// By default <see cref="GraphTypeTypeRegistry"/> maps all <see cref="int"/> .NET values to this scalar graph type.
     /// https://graphql.github.io/graphql-spec/June2018/#sec-Int
```

Comment:
```suggestion
```
I don't think we should embed links to the spec into the xml comments.  The purpose of the comment is so that people don't need the link.

Reply:
Why so? It is additional context to read about.

Reply:
Nobody does that.  I can't remember ever seeing a link in a Microsoft xml comment.  I don't think it's a good practice.  If someone wants to look up the spec, then they can look it up.

Reply:
ok

Reply:
> ok

So shall I commit the suggestion?

Reply:
I assume you're fine with the change, so I'll commit the suggestion and merge.

Reply:
I'm ok with any option.